### DISCUSSION TITLE
Arrow color

### DISCUSSION THREAD

**Discussion by CoralNi** [cdnjs/cdnjs#14093] [General] [answered]
How do I get the left and right arrows to always show and change the color to black?

**Comment by MattIPv4** [comment, marked as answer]
👋 Hi @CoralNi,

I'm not sure what exactly you are asking here, but I suspect you're looking for support with a specific library hosted on cdnjs.

Please note that cdnjs can only offer support for the CDN service itself, we do not provide support for the libraries that we host.

You will need to reach out to the library you're using itself to get support for it.

Hope that helps, Matt.
